src/appendices/git/workflows.rst

```diff
@@ -0,0 +1,170 @@
+.. _git-workflows:
+
+.. index::
+   single: Git; workflows
+
+Git Workflows
+=============
+
+This cheatsheet covers basic workflows for working with local and remote Git repositories.
+
+For details on the Git commands mentioned, or on creating and cloning repositories, revisit the chapter on :ref:`git-chapter`.
+
+.. _single-dev-branch:
+
+Single Developer / Single Branch
+--------------------------------
+
+When working on a project by yourself, you will typically follow this workflow:
+
+.. _basic-flow:
+
+#. Make changes to your code
+#. Check status: ``git status``
+#. Stage changes: ``git add .``
+#. Commit changes: ``git commit -m "Added feature X"``
+#. (If working with a remote repository) Push changes:
+   ``git push origin master``
+
+.. _team-single:
+
+Team / Single Branch
+--------------------
+
+The case of a team working on a project with a single branch is *very* uncommon. However, multiple developers working on the same branch happens quite a bit. This workflow, therefore, can be thought of as a sub-workflow of the :ref:`team-feature` workflow.
```

Comment:
If this is a sub-branch of the team/feature workflow, then it should be placed in that location on the page.  I think the progression of single to team workflows is smoother than single/team/single/team.

The order could be:
1) Single Developer/Single Branch
2) Single Developer/Feature Branches
3) Team/Single Branch (remove reference to "see stuff below")
4) Team/Feature Branches